src/Symfony/Component/Validator/Constraints/ZeroComparisonConstraintTrait.php

```diff
@@ -25,6 +25,10 @@ public function __construct(?array $options = null, ?string $message = null, ?ar
     {
         $options ??= [];
 
+        if ([] !== $options) {
```

Comment:
shouldnt this constructor also use HasNamedArguments

Reply:
indeed 👍